samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -160,7 +160,12 @@ def stdout(self):
         if self._log_file_handle:
             return self._log_file_handle
 
-        return sys.stdout
+        byte_stdout = sys.stdout
```

Comment:
can you add a comment please? It is not self-explanatory as to why this is necessary

Reply:
Done